### PR TITLE
Add default to layout

### LINKED ISSUE #80: Layout should have a default
I suggest **editor_right**.

### PR DESCRIPTION
**Pull request by aguspina**
Closes #80

### CODE REVIEW

File: app/src/editor/ExerciseController.js [hunk 1]
```diff
@@ -27,6 +27,7 @@
     $scope.hintHtml    = '';
 
     $scope.options = Bibliotheca.Options.Exercises;
+    $scope.selected.layout = $scope.selected.layout || _.first($scope.options.layouts);
```
Comment (faloi) [inline app/src/editor/ExerciseController.js]: I'd prefer to explicitly set `editor_right` as the default, perhaps also reifying that concept (the default layout) in a function.

Aside from that, why are you testing if a value is present?

Comment (aguspina) [inline app/src/editor/ExerciseController.js]: Because if the exercise exists, I don't want to change its layout

Comment (faloi) [inline app/src/editor/ExerciseController.js]: Ok, dumb question. Solve the other comment and merge it.

Comment (aguspina) [inline app/src/editor/ExerciseController.js]: Done!

**Pull request merged**